src/Controller/CocktailController.php

```diff
@@ -64,30 +51,14 @@ public function submitName(Request $request): Response
 
         $cocktailName->setCocktailName($cocktailNameEntity->getCocktailName());
 
+        $client = HttpClient::create();
+        $parsedData = new CocktailApiController($client);
```

Comment:
This is bad, research Symfony dependency injection container and implement DI logic here. New object shouldn't be created manually as I told you already.